pom.xml

```diff
@@ -150,6 +154,34 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <artifactId>maven-jar-plugin</artifactId>
+        <executions>
+          <execution>
+            <!-- Produce a jar with a platfrom-specific classifier. -->
```

Comment:
The antrun plugin generates the jar, but this execution attaches the os classifier. I should probably update the comment to read "Tag the jar with a platform-specific classifier" ?